commands/features2peptides.py

```diff
@@ -51,7 +51,7 @@
 @click.option(
     "--pnmethod",
     help="Normalization method used to normalize peptides intensities for all samples (options: mean, median, max, global, max_min, none)",
```

Comment:
interesting the default value `globalMedian` is not in the options. Please review @zprobot